### PR TITLE
Adding configuration file

### PR DESCRIPTION
**Pull request by Requiem**
Moved keybindings to a configuration file found at pydoro_core\config.py.

### CODE REVIEW

**Comment by JaDogg** [comment]
@Requiem This is really good. I have few suggestions 😊

File: pydoro/pydoro_core/config.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+bindings = dict(
```
Comment (JaDogg) [inline pydoro/pydoro_core/config.py]: This is not the right way to initialize a static dict. Use below style.
```python
DEFAULT_KEY_BINDINGS = {"focus_next": ("tab", "right", "k")...}
```

File: README.rst [hunk 2]
```diff
@@ -40,6 +40,14 @@ Usage 📖
 
 .. image:: https://github.com/JaDogg/pydoro/raw/master/images/animation.gif
 
+Configuration
```
Comment (JaDogg) [inline README.rst]: Remove this for now. 

I like your idea of maintaining set of configs internally.
Your data model is very good too. 

However I prefer to use a config file such as `~\.pydoro.ini`. That way end user doesn't need to edit code to change keybindings. And we can also introduce any new features such as different time periods to same ini file as well.

File: pydoro/pydoro_tui.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 
 from pydoro.pydoro_core.tomato import Tomato
 from pydoro.pydoro_core.util import every
+from pydoro.pydoro_core.config import bindings
```
Comment (JaDogg) [inline pydoro/pydoro_tui.py]: You will need to change the import statement accordingly.

File: pydoro/pydoro_tui.py [hunk 4]
```diff
@@ -52,14 +53,18 @@ def exit_clicked(_=None):
...
+# Key bindings. These values are set in pydoro_core/config.py.
 kb = KeyBindings()
-kb.add("tab")(focus_next)
-kb.add("s-tab")(focus_previous)
-kb.add("right")(focus_next)
-kb.add("left")(focus_previous)
-kb.add("q")(exit_clicked)
-
+for key, bind in bindings.items():
+    if(key == "focus_next"):
```
Comment (JaDogg) [inline pydoro/pydoro_tui.py]: We can get rid of this if statements if we use another dictionary here that maps function names to functions.

```python
ACTIONS = {"focus_next": focus_next, ...}
```

In python functions are objects too, so we can assign them as a value in a dict.

**Comment by JaDogg** [comment]
@Requiem if you need any help with the requested changes let me know 😊

**Comment by JaDogg** [comment]
Since I didn't get any response, I'm merging this. 👍 I'll apply the changes on top

**Comment by Requiem** [comment]
Shoot, I'm sorry @JaDogg, I got busy and forgot to send a reply. I agree with your suggestions but haven't had a chance to look at implementing them yet. I can take a go at it early next week though.

**Comment by JaDogg** [comment]
@Requiem No problem. I actually edited it myself lol 😂

**Pull request merged**